exporter/exporterhelper/queued_retry_inmemory.go

```diff
@@ -12,11 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//go:build !enable_unstable
-// +build !enable_unstable
-
 package exporterhelper // import "go.opentelemetry.io/collector/exporter/exporterhelper"
 
+// TODO: rename the file to queued_retry_sender.go after merging the PR
```

Comment:
Why this TODO? Why can't we rename before merging the PR?

Reply:
@tigrannajaryan this is due to [this suggestion](https://github.com/open-telemetry/opentelemetry-collector/pull/5457#pullrequestreview-994187835)